.env.example

```
@@ -4,8 +4,8 @@
 # The ports intentionally avoid Supabase CLI defaults so another local Supabase
 # project can keep running on 54320-54329.
 
-NEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54331
-NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=replace-with-local-publishable-key-from-pnpm-supabase-status
+BETTER_AUTH_URL=http://localhost:3000
+BETTER_AUTH_SECRET=replace-with-a-32-character-or-longer-local-secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the BETTER_AUTH_URL key

(UnorderedKey)